R/bootstrap.nlsfit.R

```diff
@@ -1044,3 +1098,43 @@ plot.bootstrapfit <- function(x, ..., col.line="black", col.band="gray", opacity
   ## plot the fitted curve on top
   lines(x=X, y=Y, col=col.line, lty=lty, lwd=lwd)
 }
+
+residual_plot <- function (x, ...) {
+  UseMethod("residual_plot", x)
```

Comment:
```suggestion
  UseMethod("residual_plot", x, ...)
```

Reply:
It works without that, I believe that `UseMethod` magically forwards all the arguments.

Reply:
I don't see how this is possible and would add the dots in order to avoid this confusion in future. But do as you like.

Reply:
`UseMethod` only takes two arguments, the dots will be ignored anyway. Under the hood, magic happens and a function call is generated which matches the signature of the generic, which has `x` as its first argument and `...` as its second. It's really weird... see *Technical Details* in `?UseMethod`